src/helpers/ExecuteEvents.js

```diff
@@ -0,0 +1,20 @@
+import ScriptRunner from 'src/helpers/ScriptRunner';
```

Comment:
Any particular reason to remove it from Container file and creating a separate file? Do you expect more such functions to come in? If so, maybe better name than a verb (executeEvents.js)

Reply:
Moved to a separate file since it has no dependency with the container.

Reply:
As per the convention being followed the file should be in Camel Case and not Pascal case, please chane

Reply:
Updated.